client/my-sites/plans-v2/plans-column/index.tsx

```diff
@@ -66,6 +66,7 @@ const PlanComponent = ( {
 			onButtonClick={ () => onClick( plan ) }
 			features={ { items: [] } }
 			originalPrice={ price }
+			withStartingPrice={ Array.isArray( plan.subtypes ) && plan.subtypes.length > 0 }
```

Comment:
Nitpick: this level of defensive coding is unneeded, all `SelectorProduct` types have `subtypes` as an array with the most recent merge.